clouddriver-docker/src/test/java/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClientTest.java

```diff
@@ -202,4 +202,33 @@ public void getCatalogWithNextLink() {
     DockerRegistryCatalog dockerRegistryCatalog = dockerRegistryClient.getCatalog();
     assertEquals(15, dockerRegistryCatalog.getRepositories().size());
   }
+
+  @Test
+  public void getTagsWithNextLinkEncryptedAndEncoded() {
+    wmDockerRegistry.stubFor(
+        WireMock.get(urlMatching("/v2/library/nginx/tags/list\\?n=5"))
+            .willReturn(
+                aResponse()
+                    .withStatus(HttpStatus.OK.value())
+                    .withHeader(
+                        "link",
+                        "</v2/library/nginx/tags/list?last=Md1Woj%2FNOhjepFq7kPAr%2FEw%2FYxfcJoH9N52%2Blo3qAQ%3D%3D&n=5>; rel=\"next\"")
+                    .withBody(tagsResponseString)));
+
+    wmDockerRegistry.stubFor(
+        WireMock.get(
+                urlMatching(
+                    "/v2/library/nginx/tags/list\\?last=Md1Woj%2FNOhjepFq7kPAr%2FEw%2FYxfcJoH9N52%2Blo3qAQ%3D%3D&n=5"))
+            .willReturn(
+                aResponse().withStatus(HttpStatus.OK.value()).withBody(tagsSecondResponseString)));
+
+    wmDockerRegistry.stubFor(
+        // notice %252F, %253D which are double encoded characters of /(or %2F) and =(or %3D)
+        WireMock.get(
+                urlMatching(
+                    "/v2/library/nginx/tags/list\\?last=Md1Woj%252FNOhjepFq7kPAr%252FEw%252FYxfcJoH9N52%252Blo3qAQ%253D%253D&n=5"))
+            .willReturn(aResponse().withStatus(405).withBody("Method Not Allowed")));
```

Comment:
Even with the fixes, does clouddriver make a call that results in a 405?

Reply:
Hmmm...maybe this will be more clear with appropriate verify calls.  I'm pretty tempted to take it out as part of the "fix" commit though.

Reply:
Added verify calls and removed 405 stub.